lib/Connection.js

```diff
@@ -37,6 +40,20 @@ function Connection(router, req, res) {
             }
         }
 
+        // Cache similar GET requests
```

Comment:
e.g. `/foo?bar=1&props=a,b,c` would match `/foo?props=a,b,c&bar=1`, but not match `/foo?bar=1&props=c,b,a` unless I sorted the props.